client/mm/libxc/binance.go

```diff
@@ -1604,8 +1777,12 @@ func (bnc *binance) connectToMarketDataStream(ctx context.Context, baseID, quote
 			PingWait: time.Minute * 4,
 			ReconnectSync: func() {
 				bnc.log.Debugf("Binance reconnected")
+				select {
+				case reconnectC <- struct{}{}:
+				default:
+				}
```

Comment:
What's the reason for this?

Reply:
I'm not sure. I think this is a batch of changes I got from @buck54321 

A natural reconnect shouldn't need a new connection I guess.